ebml_matroska.xml

```diff
@@ -63,7 +63,7 @@
       </element>
       TimecodeScale When combined with <a href="http://www.matroska.org/technical/specs/index.html#TimecodeScaleDenominator">TimecodeScaleDenominator</a> the Timestamp scale is given by the fraction TimecodeScale/TimecodeScaleDenominator in seconds.-->
       <element name="Duration" level="2" id="0x4489" type="float" minver="1" range="&gt; 0x0p+0">
-        <documentation lang="en">Duration of the Segment (based on TimecodeScale).</documentation>
+        <documentation lang="en" type="definition">Duration of the Segment measured in nanoseconds when scaled by TimecodeScale.</documentation>
```

Comment:
Also an issue that TimecodeScale's definition refers to the content with `timestamp` terms.